values/src/main/java/org/spine3/net/UrlPrinter.java

```diff
@@ -48,7 +48,6 @@ static String printToString(Url url) {
             return url.getRaw();
         }
 
-        //noinspection StringBufferWithoutInitialCapacity
         // We don't know the capacity at this point
         final StringBuilder sb = new StringBuilder();
```

Comment:
Isn't this still an issue? If `IDEA` highlights this e.g. as a _warning_, please suppress with `@Suppress("...")` annotation.